ui/src/app/pages/workspace/workspace-wrapper/component.html

```diff
@@ -29,7 +29,7 @@ <h3 class="modal-title">Error:</h3>
                     [shareFunction]="handleShareAction"
                     [sidebarOpen]="sidebarOpen"
                     [notebookStyles]="notebookStyles"></app-help-sidebar>
-  <div [style.margin-right]="helpContent ? '45px' : 0" [style.height]="helpContent ? 'auto': '100%'">
+  <div [style.margin-right]="helpContent ? '45px' : 0" [style.height]="helpContent && !notebookRouterOutletContainerStyles ? 'auto': '100%'">
```

Comment:
Please add a comment somewhere to explain what this is doing, or else rename property to be more clearly tied to the display implications per above comment